content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md

```diff
@@ -387,210 +333,78 @@ To compile and start the updated Substrate node:
 
 1. Compile the node template by running the following command:
 
-   ```bash
-   cargo build --release
-   ```
+  ```bash
+  cargo build --release
+  ```
 
 1. Start the node in development mode by running the following command:
 
-   ```bash
-   ./target/release/node-template --dev
-   ```
+  ```bash
+  ./target/release/node-template --dev
+  ```
 
-   The `--dev` option starts the node using the predefined `development` chain specification.
-   Using the `--dev` option ensures that you have a clean working state any time you stop and restart the node.
+  The `--dev` option starts the node using the predefined `development` chain specification.
+  Using the `--dev` option ensures that you have a clean working state any time you stop and restart the node.
 
 1. Verify the node produces blocks.
 
-## Build a custom front-end component
-
-Now that you have a new blockchain running with the custom proof-of-existence pallet, let's add a custom React component to the front-end template.
-This React component enables you to expose the proof-of-existence capabilities and interact with the new pallet you created.
```

Comment:
Totally removed this Front-End React JS copy paste, and replaced it with the Polkadot JS Apps.

Much better experience I think.